content/docs/hooks-faq.md

```diff
@@ -18,102 +20,102 @@ This page answers some of the frequently asked questions about [Hooks](/docs/hoo
 
-### What do Hooks mean for popular APIs like Redux `connect()` and React Router? {#what-do-hooks-mean-for-popular-apis-like-redux-connect-and-react-router}
+### ¿Qué significan los Hooks para APIs populares como el connect de Redux, o React Router? {#what-do-hooks-mean-for-popular-apis-like-redux-connect-and-react-router}
 
-You can continue to use the exact same APIs as you always have; they'll continue to work.
+Puedes seguir usando exactamente las mismas APIs que siempre has usado, seguirán funcionando.
 
-In the future, new versions of these libraries might also export custom Hooks such as `useRedux()` or `useRouter()` that let you use the same features without needing wrapper components.
+En el futuro, nuevas versiones de estas librerías también podrían exportar Hooks personalizados como `useRedux()` or `useRouter()`, que te permitan usar las mismas características sin necesidad de usar componentes que los envuelvan.
 
-### Do Hooks work with static typing? {#do-hooks-work-with-static-typing}
+### ¿Funcionan los Hooks con tipado estático? {#do-hooks-work-with-static-typing}
 
-Hooks were designed with static typing in mind. Because they're functions, they are easier to type correctly than patterns like higher-order components. The latest Flow and TypeScript React definitions include support for React Hooks.
+Los Hooks fueron diseñados con el tipado estático en mente. Al ser funciones, son más fáciles de tipar que patrones como los componentes de alto orden (HOC). Las últimas definiciones para React de TypeScript y Flow incluyen soporte para Hooks.
 
-Importantly, custom Hooks give you the power to constrain React API if you'd like to type them more strictly in some way. React gives you the primitives, but you can combine them in different ways than what we provide out of the box.
+Aún más importante, los Hooks personalizados tienen el poder de restringir la API de React si quisieras tiparlas de una manera más estricta. React te da las primitivas, pero puedes combinarlas de distintas maneras de las que proveemos por defecto.
 
-### How to test components that use Hooks? {#how-to-test-components-that-use-hooks}
+### ¿Cómo testear Componentes que usan Hooks? {#how-to-test-components-that-use-hooks}
 
-From React's point of view, a component using Hooks is just a regular component. If your testing solution doesn't rely on React internals, testing components with Hooks shouldn't be different from how you normally test components.
+Desde el punto de vista de React, un componente que use Hooks, sigue siendo un componente normal. Si las herramientas de testing que utilizas no depende de los mecanismos internos de React, testear los componentes que usen Hooks, no debería ser diferente de testear cualquier otro componente.
 
-For example, let's say we have this counter component:
+Por ejemplo, asumamos que tenemos este componente de Conteo:
```

Comment:
I suggest `componente contador`.

Reply:
I'm not that sure about this one, they convey the same meaning, though I think specifying it in terms of what the component does is better than specifying it in terms of what the component is

Reply:
Agree with @caal-15, as they mean the same and it's proper Spanish. 👍